editor/src/core/shared/project-contents-utils.ts

```diff
@@ -0,0 +1,58 @@
+import { objectKeyParser, parseString } from '../../utils/value-parser-utils'
+import { defaultEither, eitherToMaybe, isLeft } from './either'
+import { is } from './equality-utils'
+import { memoize } from './memoize'
+import { isCodeFile, ProjectContents, ProjectFile } from './project-file-types'
+
+function parsePackageJsonInner(packageJson: ProjectFile | null): Record<string, any> | null {
```

Comment:
Strictly I think the parameter type should include `| undefined`, as that's what you get if you key into an object.

Reply:
Done